assets/airports/egnm.json

```diff
@@ -10,251 +10,2015 @@
+                "32": [["_ILF0.5DME", "A12+"], "_ILF2.1DME", "_ILF3.5DME", ["_INTGAM319", "A30+|A70-"], ["_GAM319038", "A35+|A70-"], ["_GAM319035", "A60+|A70-"], ["_GAM319031", "A70"], "_GAM319019", "_GAM319018", "_GAMR319INTPOLR118", "_GAM15DME"]
             },
-            "body": [],
+            "body": ["DOPEK"],
             "exitPoints": {
-                "NELSA": [["NELSA", "A70"]]
+                "LAMIX": ["LAMIX"]
             },
             "draw": [
-                ["05DME", "21DME", "35DME", "D052L", "D052K", "D272J", "NELSA"]
+                ["DOPEK", "LAMIX"]
             ]
         },
-        "DOPEK": {
-            "icao": "DOPEK",
-            "name": "Dopek",
-            "suffix": {"32":"2W", "14":"2X"},
+        "NELSA3W": {
+            "icao": "NELSA3W",
+            "name": "Nelsa Three Whiskey",
             "rwy": {
-                "32": ["05DME", "21DME", "35DME", ["R255", "A30"], ["GAM38", "A35"], ["GAM35", "A60"], ["GAM31", "A70"], "GAM19", "GAM18", "GAM15"],
-                "14": ["D140B", ["GAM36", "A30"], ["GAM32", "A35"], ["GAM29", "A70"], "GA19A", "GA15A", "GAM14"]
+                "32": [["_ILF0.5DME", "A12+"]]
             },
-            "body": [],
+            "body": ["_ILF2.1DME", "_ILF3.5DME", ["_POL053012", "A30+|A70-"], "_POL053011" ,["_POL053010", "A35+|A70-"], "_INTLBA272"],
             "exitPoints": {
-                "DOPEK": ["DOPEK"]
+                "NELSA": [["NELSA", "A70"]]
             },
-            "draw": [
-                ["35DME", "R255", "GAM38", "GAM35", "GAM31", "GAM19", "GAM18", "GAM15", "DOPEK"],
-                ["D140B", "GAM36", "GAM32", "GAM29", "GA19A", "GA15A", "GAM14", "DOPEK"]
-            ]
+            "draw": [["_ILF0.5DME", "_ILF2.1DME", "_ILF3.5DME", "_POL053012", "_POL053011", "_POL053010", "_INTLBA272", "NELSA"]]   
         },
-        "LAMIX": {
-            "icao": "LAMIX",
-            "name": "Lamix",
-            "suffix": {"32":"2W", "14":"2X"},
+        "POL2X": {
+            "icao": "POL2X",
+            "name": "Pole Hill Two Xray",
             "rwy": {
-                "32": ["05DME", "21DME", "35DME", ["R255", "A30"], ["GAM38", "A35"], ["GAM35", "A60"], ["GAM31", "A70"], "GAM19", "GAM18", "GAM15", "DOPEK"],
-                "14": ["D140B", ["GAM36", "A30"], ["GAM32", "A35"], ["GAM29", "A70"], "GA19A", "GA15A", "GAM14", "DOPEK"]
+                "14": ["_ILBF2DME"]
             },
-            "body": [],
+            "body": [["_POL083014", "A30+|A70-"], ["_POL083010", "A35+|A70-"]],
             "exitPoints": {
-                "LAMIX": ["LAMIX"]
+                "POL": [["POL", "A70"]]
             },
-            "draw": [
-                ["DOPEK", "LAMIX"]
-            ]
+            "draw": [["_ILBF2DME", "_POL083014", "_POL083010", "POL"]]
         }
     },
     "spawnPatterns": [
```

Comment:
There's currently about 1 aircraft every 10 minutes, and while this is perfectly realistic, it might become very boring to play at Leeds. If we increase this to 10-15acph it'll be slightly more challenging, while still within the limits of what we consider realistic.

Reply:
Roger that 👍